src/components/calcite-flow-item/calcite-flow-item.tsx

```diff
@@ -52,6 +53,11 @@ export class CalciteFlowItem {
    */
   @Prop() showBackButton = false;
 
+  /**
+   * Summary text.
```

Comment:
pebble: Needs better description. Curious what the subtle difference between heading and summary are, as a summary can act as a heading like it does in our PRs sometimes.